x-pack/test/functional/page_objects/index_management_page.ts

```diff
@@ -144,9 +144,11 @@ export function IndexManagementPageProvider({ getService }: FtrProviderContext)
       await testSubjects.setValue('createIndexNameFieldText', value);
     },
     async clickCreateIndexSaveButton() {
-      await testSubjects.click('createIndexSaveButton');
-      // Wait for modal to close
-      await testSubjects.missingOrFail('createIndexSaveButton');
+      await retry.tryForTime(60_000, async function waitForClickAndClose() {
+        await testSubjects.click('createIndexSaveButton');
+        // Wait for modal to close
+        await testSubjects.missingOrFail('createIndexSaveButton');
+      });
```

Comment:
From the failure screenshot it looks like the modal just takes a long while to go away. This isn't about the click not coming through. So I'd suggest to not retry with the click, but just increase the timeout on the `missingOrFail` call. WHat do you think?

Reply:
Ahh roger that, I see what you mean boss man.  I'll update shortly.

Reply:
https://github.com/elastic/kibana/pull/185897/commits/d0f3179bf2452d6fe1c5e46d54bffe02f4edfb6d